site/components/src/main/java/uk/nhs/digital/ps/components/PublicationComponent.java

```diff
@@ -57,7 +59,8 @@ public void doBeforeRender(final HstRequest request, final HstResponse response)
             index.add(KEY_FACTS_ID);
         }
 
-        if (isNotBlank(publication.getAdministrativeSources())) {
+        // TODO remove publication check in DW-1285
```

Comment:
Did you mean to commit this todo message?

Reply:
It should be fixed now.